sql/src/test/java/io/crate/integrationtests/ClientNodeIntegrationTest.java

```diff
@@ -40,6 +40,11 @@ public Client client() {
                         // make sure we use a client node (started with client=true)
                         return internalCluster().clientNodeClient();
                     }
+
+                    @Override
```

Comment:
Nullable is in the interface definition and this is an anonymous implementation, so any consumer uses the Interface and sees the Nullable annotation.